packages/i18n/README.md

```diff
@@ -0,0 +1,76 @@
+i18n
+======
+
+Internationalization utilities for client-side localization.
+
+https://codex.wordpress.org/I18n_for_WordPress_Developers
+
+
+## Installation
+
+Install the module:
+
+```bash
+npm install @wordpress/i18n --save
+```
+
+```js
+import { sprintf, _n } from '@wordpress/i18n';
+
+sprintf( _n( '%d hat', '%d hats', 4, 'text-domain' ), 4 );
+// 4 hats
+```
+
+Note that you will not need to specify [domain](https://codex.wordpress.org/I18n_for_WordPress_Developers#Text_Domains) for the strings.
+
+## Build
+
+You can use the [WordPress i18n babel plugin](../babel-plugin-makepot/README.md) to generate a `.pot` file containing all your localized strings.
+
+The package also includes a `pot-to-php` script used to generate a php files containing the messages listed in a `.pot` file. This is usefull to trick WordPress.org translation strings discovery since at the moment, WordPress.org is not capable of parsing strings directly from JavaScript files.
+
+```sh
+node tools/pot-to-php languages/myplugin.pot languages/myplugin-translations.php text-domain
```

Comment:
I think we want to make this a proper `bin` entry in `package.json`:

https://docs.npmjs.com/files/package.json#bin

That way, a developer could execute via `npx pot-to-php`.

As-is, the code snippet isn't very usable (would need to at least be prefixed with `./node_modules/@wordpress/i18n`).